spec/requests/proxying_spec.rb

```diff
@@ -91,6 +91,35 @@
         end
       end
     end
+
+    context "with a JWT token in a cookie" do
+      let(:cookie) { "auth_bypass_token=#{token}; Path=/; Domain=#{'.' + upstream_uri}" }
+
+      before do
+        allow_any_instance_of(Proxy).to receive(:jwt_auth_secret).and_return(jwt_auth_secret)
```

Comment:
I think we do this in most places so might want to move this up to a shared before.